workspace/utilities/site-variables.xsl

```diff
@@ -1,13 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:exslt="http://exslt.org/common"
+	exclude-result-prefixes="exslt">
 
-	<!-- Default site name with current language -->
-	<xsl:variable name="site-name">
-		<xsl:call-template name="default-site-name" />
-	</xsl:variable>
+<xsl:variable name="data" select="/data" />
+<xsl:variable name="params" select="$data/params" />
```

Comment:
Same here.

Reply:
I think it's a "cleaner" way to access the data. That way all our pointers inside the various templates starts with `$` and not a mix of `$` & `/`.

Reply:
Yes, but mainly for the reason @hotdoy mentions above. When we create a RSL, we lose access to the parents and its can break things or just make things harder (think Date template, who needs /data )